build.gradle

```diff
@@ -93,6 +97,7 @@ dependencies {
 
 	// Annotations
 	implementation group: 'org.jetbrains', name: 'annotations', version: "${project.jetbrains_annotations}"
+	implementation 'org.jetbrains:annotations:15.0'
```

Comment:
Were the annotations not working when you added this?

Reply:
Okay so the missing texture box didn't work but I fixed that issue, I'll finish up with checkstyle and other issues and then merge it